units/en/unit3/customize.mdx

```diff
@@ -0,0 +1,2 @@
+# Customize your own demo
+
```

Comment:
Would also be cool to have a call-for-action for users to upload their creations as demo Spaces.

Reply:
They will, to do the demo they need to duplicate the space